### ISSUE TITLE
default export is not correct

### ISSUE THREAD

**Issue by dmackerman** [divriots/style-dictionary-to-figma#7]
Hello!

I tried importing the code in this way:

`const styleDictionaryToFigma = require('@divriots/style-dictionary-to-figma');`

But, found that it doesn't work. Needed to add `.default`.

`const styleDictionaryToFigma = require('@divriots/style-dictionary-to-figma').default;`

Also may I suggest you change the README to use the `registerTransform` method on StyleDictionary:

```js
StyleDictionaryPackage.registerFormat({
  name: 'figmaTokensPlugin',
  formatter({ dictionary }) {
    const parsedTokens = styleDictionaryToFigma(dictionary.tokens);
    return JSON.stringify(parsedTokens, null, 2);
  }
});
```

Thanks!

**Comment by jorenbroekema** [comment]
This is only true for the CommonJS entrypoint of this library, for ESM (modern way, which I recommend), the import is correct.

Perhaps it makes sense to not use default exports in combination with named exports, since this only works in ESM and not in CJS, meaning you get a slightly different API for CJS because it requires the `.default` prop. There's other issues with default exports as well that I was recently made aware of, imo making it an anti-pattern.

I'll do a breaking update on this lib that will straighten things out:

```js
import { transform } from '@divriots/style-dictionary-to-figma';
const { transform } = require('@divriots/style-dictionary-to-figma');
```

As for your README suggestion, both ways work, either by registerFormat or in the format prop in the config, I just prefer doing everything in the config. I'll clarify it in the readme because I do think registerFormat is the one people are more acquainted with.

**Comment by jorenbroekema** [comment]
[Fixed in 0.2.0](https://github.com/divriots/style-dictionary-to-figma/blob/main/CHANGELOG.md#020)
